communicationPlugIn-Kafka/src/main/java/com/cgi/bonnie/configuration/TopicConfig.java

```diff
@@ -40,14 +36,11 @@ public NewTopic bonnieMessageTopic() {
         return TopicBuilder.name(topicMessage).build();
     }
 
+    //fixme  there need only 2 topic -> one bean is not necessary. This property is already defined in KafkaMessageService class, but there <String,String>
```

Comment:
@abalazsik    it is a remaining part from merging the 2 kafka implementation, but one of them already contined the others